tests/system/feature_store/test_spark_engine.py

```diff
@@ -1362,7 +1362,7 @@ def test_get_offline_features_with_filter_and_indexes(self, timestamp_key):
             target=target,
             query="bad>6 and bad<8",
             engine="spark",
-            run_config=fstore.RunConfig(local=self.run_local),
+            run_config=fstore.RunConfig(local=self.run_local, kind="remote-spark"),
```

Comment:
Please merge from development so that these don't appear in your diff anymore (and because it's nice).